rockcraft/extensions/fastapi.py

```diff
@@ -265,7 +265,13 @@ def _asgi_entrypoint_error_messages(self) -> list[str]:
         try:
             self._find_asgi_location()
         except FileNotFoundError:
-            return ["missing ASGI entrypoint"]
+            return [
+                ("missing ASGI entrypoint\n"
+                "  We looked for an `app` global variable in the following places:\n"
+                "  1. `app.py`.\n"
+                "  2. Inside the directories `app`, `src` and rockcraft name, in the files"
+                "`__init__.py`, `app.py` or `main.py`." ),
```

Comment:
```suggestion
                "  Cannot find 'app' global variable in:\n"
                "  1. app.py\n"
                "  2. In files __init__.py, and main.py in the 'app',
                "'src', and root project directories."),
```

It's confusing to me that we print `app.py` twice. I might be misunderstanding point # 2. Please correct me if it's wrong. :)